template/src/App.vue

```diff
@@ -26,6 +26,11 @@
 import Hello from './components/Hello'{{#if_eq lintConfig "airbnb"}};{{/if_eq}}
 
 export default {
+  data () {
+    return {
+      logo: require('./assets/logo.png')
+    }
+  },
   components: {
```

Comment:
The changes in this file should no longer be necessary. They were a workaround for a bug that used to exist in vue-loader, but it's been fixed now.